### PR TITLE
Add tests and docs

### PR DESCRIPTION
**Pull request by dtitov**
It's more of int-tests actually, because I spawn embedded web-server to fake CEGA response and also spawn embedded RabbitMQ broker to validate that messages were successfully published. But it doesn't run Docker environment - everything happens on Java/Maven level, so it's more lightweight than what we have in the test-suite of the main repo (so, all you need is `mvn test` 😃).

### CODE REVIEW

**Comment by blankdots** [comment]
Seems that `embeddedRabbitMq` requires Erlang installed (not really an issue) but maybe we should look at what call it makes to RabbitMQ, catch those calls and mock the expected responses.
With the web server seems [mockito](http://site.mockito.org/) is a good option https://specto.io/blog/2017/7/7/mocking-http-mockito-style/
Let us keep them unit-tests level.
@juhtornr please help a bit with Java code.

**Comment by dtitov** [comment]
According to the comments, embedded web-server and embedded RabbitMQ broker are replaced with Mockito mocks. Now it's pure unit-tests.

**Comment by blankdots** [comment]
One more suggestion so we can have somewhat a similar environment on all LocalEGA projects, coverage reports (and the cool coverage badge) :name_badge: :
* https://www.eclemma.org/jacoco/trunk/doc/maven.html - maybe there are other
* we use coveralls for python code (feel free to chose smth else) ... https://github.com/trautonen/coveralls-maven-plugin - this should only send reports after run on travis (that is when someone enables the project), for running the tests on local we don't need them

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-SFTP Inbox for LocalEGA
\ No newline at end of file
+# SFTP Inbox for LocalEGA
+
+Inbox login system
```
Comment (blankdots) [inline README.md]: don't think we should keep this line: `Inbox login system` it is fine just having the title `# SFTP Inbox for LocalEGA` - sry a bit nitpicking

Comment (dtitov) [reply README.md]: I just copied it from ReadTheDocs. But I see your point. I've updated formatting to make it a sub-header.

**Comment by blankdots** [comment]
LGTM, but I cannot merge :disappointed:

**Comment by juhtornr** [comment]
Me neither :(

**Comment by blankdots** [comment]
@dtitov should we merge like this, or wait for the Quality Review to pass ?

**Comment by dtitov** [comment]
@blankdots, actually, I was just playing with Codacy and didn’t expect it to insert any checks to GitHub. I already turned it off, but the check is still here. So, I guess, we can merge: those warnings are not completely relevant. And anyway it gives the “A” grade to the code :D

**Pull request merged**